camunda/v2/client.go

```diff
@@ -153,3 +154,57 @@ func (c *client) doPostRequest(ctx context.Context, params *bytes.Buffer, endpoi
 
 	return body, nil
 }
+
+func (c *client) DeleteTask(ctx context.Context, businessKey string) error {
+	tasks, err := c.getTasks(ctx, businessKey)
+	if err != nil {
+		return err
+	}
+	if len(tasks) == 1 {
+		if err = c.deleteProcessInstance(ctx, tasks[0].ProcessInstanceId); err != nil {
+			return err
+		}
+	} else {
+		return fmt.Errorf("found %d camunda tasks for businessKey: %s", len(tasks), businessKey)
+	}
+	return nil
```

Comment:
maybe if we first validate the error cases the logic becomes a little more easier to read, wdyt?

```suggestion
	if len(tasks) != 1 {
		return fmt.Errorf("found %d camunda tasks for businessKey: %s", len(tasks), businessKey)
	}
	
	return c.deleteProcessInstance(ctx, tasks[0].ProcessInstanceId)
```

Reply:
done